lua/kubectl/mappings.lua

```diff
@@ -41,5 +48,84 @@ function M.register()
       vim.api.nvim_input("R")
     end,
   })
+
+  vim.api.nvim_buf_set_keymap(0, "n", "1", "", {
+    noremap = true,
+    silent = true,
+    desc = "Deployments",
+    callback = function()
+      deployments_view.Deployments()
+    end,
+  })
+
+  vim.api.nvim_buf_set_keymap(0, "n", "2", "", {
+    noremap = true,
+    silent = true,
+    desc = "Pods",
+    callback = function()
+      pods_view.Pods()
+    end,
+  })
+
+  vim.api.nvim_buf_set_keymap(0, "n", "3", "", {
+    noremap = true,
+    silent = true,
+    desc = "Configmaps",
+    callback = function()
+      configmaps_view.Configmaps()
+    end,
+  })
+
+  vim.api.nvim_buf_set_keymap(0, "n", "4", "", {
+    noremap = true,
+    silent = true,
+    desc = "Secrets",
+    callback = function()
+      secrets_view.Secrets()
+    end,
+  })
+
+  vim.api.nvim_buf_set_keymap(0, "n", "5", "", {
+    noremap = true,
+    silent = true,
+    desc = "Services",
+    callback = function()
+      services_view.Services()
+    end,
+  })
+
+  vim.api.nvim_buf_set_keymap(0, "n", "g?", "", {
+    noremap = true,
```

Comment:
We should probably do a merge of view specific + globals for the g?

Reply:
I had specific hints g? for each view before migrating them to mappings.lua, but what exactly do you have in mind?

Reply:
Similar to the normal hints section where we have view specific things and then we add global hints.
So for the pod specific view it would look like:
pod specific: [logs | desc | top | enter] - [1 pod | 2 dep | 3 configmaps]
service specific: [desc | port-forward] - [1 pod | 2 dep 3 confimaps]

Those globals would be added for every view but, I think it's really helpful if we could have contextual hints (similar to neogit)

Reply:
I just checked the mapping for each view, and they're already showing their specific mappings under g?.

I think the next step is to add more functionality and later populate the g? for more hints if necessary. I think we can merge this PR since the implementation is in-scope.

Reply:
Yeah, they get replaced by the global one but I agree that this PR has nothing to do with that, I'll create another PR that addresses that part, great work on the numbering shortcuts! I'm already using them 👍